metrics/web/lib/auth/presentation/state/auth_notifier.dart

```diff
@@ -119,7 +120,11 @@ class AuthNotifier extends ChangeNotifier {
         assert(_updateUserProfileUseCase != null);
 
   /// Determines if a user is authenticated.
-  bool get isLoggedIn => _isLoggedIn;
+  bool get isLoggedIn =>
+      _authState != null && _authState != AuthState.loggedOut;
```

Comment:
```suggestion
  bool get isLoggedIn =>
      _authState == AuthState. loggedIn || _authState == AuthState.loggedInAnonymously;
```
Suggest to go this route - to make sure that if we add additional not logged in state it will not be automatically treated as loggedIn.